src/mmw/npm-shrinkwrap.json

```diff
@@ -44,6 +44,11 @@
       "from": "https://registry.npmjs.org/bootstrap/-/bootstrap-3.3.4.tgz",
       "resolved": "https://registry.npmjs.org/bootstrap/-/bootstrap-3.3.4.tgz"
     },
+    "bootstrap-datepicker": {
```

Comment:
👏  I like the neatness of the shrinkwrap changes! There is a typo in the commit message though `javascrip` → `javascript`.